models/rentals.js

```diff
@@ -0,0 +1,23 @@
+"use strict";
+
+var sqlite3 = require('sqlite3').verbose();
+
+function Rental() {
+  this.tableName = "rentals";
+}
+
+Rental.prototype = require('./database').prototype;
+
+Rental.prototype.return = function update(movie_title, date, callback) {
```

Comment:
The function names don't match anymore ;)

(Also, there are now merge conflicts)